components/atoms/icon/icon/__tests__/stories.js

```diff
@@ -6,11 +6,19 @@
 import React from 'react'
 
 // Storybook
-import { boolean, color, object, text } from '@storybook/addon-knobs'
-import { Context, IconPrefix, IconSize, ListIcons, Wrapper } from 'decorators'
+import { Wrapper } from 'decorators'
```

Comment:
@mhsnprvr3 Question can we make this default on every story - in the mian / preview.js - I am sure we can